examples/ping.py

```diff
@@ -0,0 +1,7 @@
+Get-Content "api.conf" | foreach-object -begin {$conf=@{}} -process { $fields = [regex]::split($_,' = ');
```

Comment:
For me this has to be one of these : 
- A library method
- call for a powershell module that does the same and is a dependency of our library

But i don't think that we want specific code inside of each example. I'm checking for a powershell module equivalent to what we do in python.

Reply:
Can make it so that this piece of code is used in a -file option when calling the library

then we would do something like : 

```$client = Get-CyberwatchApi -conf api.conf```

Reply:
This would be nice !